packages/node-core/src/utils/dictionaryEntry.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {Text} from '@polkadot/types-codec';
+import {DictionaryQueryEntry} from '@subql/types';
+import {Dictionary} from '../indexer/dictionary.service';
+// import { SubqlProjectDs } from '../../../node/src/configure/SubqueryProject';
+
+export function buildDictionaryEntryMap(
```

Comment:
I'd like to see these functions as a class, and we can maintain the `mappedDictionaryQueryEntries` in the class. It would be good to clean up the args to these functions too